jenkins-integration/jenkins-jobs/common/scripts/background/curl-server-metrics-loop.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -x
+
+while true ; do
```

Comment:
For prod, I don't think it will be a problem.  For dev, I think it might be confusing if we end up graphing data from multiple runs together for one job instance.  I understand the concern about potentially losing data if we just wipe it out before we start.  Maybe a compromise would be to move the file at the start of a run, if it exists, to a timestamped backup - like `metrics.json.20160826...`? That way, each file would have a set of metrics for a single job run only.